File: demo/src/com/day2/Test1.java
package com.day2;

public class Test1 {

	public static void main(String[] args) {

		float f = 0;
		double d = 0;
		
		//for(시작값;최대값;증가값)
		for(int i=1;i<=100000;i++) { //i=i+1
			
			f+=100000;	//f = f + 100000; 
			d+=100000;	//d = d + 100000;
			
		}
		
		System.out.println("float: " + (f/100000));
		System.out.println("double: " + (d/100000));
		
	}

}

File: React_basic/test6/src/components/Test1.js
import React, { useState } from 'react';
// 고유번호 처리방법
const Test1 = () => {

    const [data,setData] = useState([])
    //{id:1, name:배수지00}

    const onAdd = () => {
        setData(data.concat({id:data.length,
        name:'배수지' + Math.floor(Math.random()*10)}))
    }

    return (
        <div>
            <button onClick={onAdd}>데이터 추가</button>
            {
                data.map(item => <p key={item.id}>
                    {item.id} / {item.name}
                </p>)
            }
        </div>
    );
};

export default Test1;